.golangci.yml

```diff
@@ -0,0 +1,10 @@
+linters:
+  disable-all: true
+  enable:
+  - gofmt
+  - govet
+  - revive
```

Comment:
These are the pre-existing linters that we were running in the make file (revive is the successor to golint).

Reply:
Over time we can go ahead and add additional linters as well, which would help highlight issues like in #232 where you caught an issue with my `UpdateFunc` in review, where I wasn't using `old` and instead was using `obj` twice 😬

Reply:
😄